src/services/qix/session.js

```diff
@@ -11,17 +13,22 @@ let connectionIdCounter = 0;
 class Session {
 
   /**
-  * Constructor
-  * @param {Object} rpc - The RPC instance used by the session.
-  * @param {Boolean} delta=true - Flag to determine delta handling.
-  * @param {Object} definition - The definition instance used by the session.
-  * @param {Object} JSONPatch - JSON patch object.
-  * @param {Object} listeners A key-value map of listeners.
-  * @param {Array} interceptors An array of interceptors.
-  * @param {Function} Promise - The promise constructor.
-  */
+   * Constructor
```

Comment:
We should create an issue to refactor these parameters (passing in the config object directly).